OpenRA.Platforms.Default/Sdl2Input.cs

```diff
@@ -92,6 +92,14 @@ public void PumpInput(Sdl2PlatformWindow device, IInputHandler inputHandler, int
 								case SDL.SDL_WindowEventID.SDL_WINDOWEVENT_SIZE_CHANGED:
 									device.WindowSizeChanged();
 									break;
+
+								case SDL.SDL_WindowEventID.SDL_WINDOWEVENT_MINIMIZED:
+									Game.GameWindowSuspended = true;
```

Comment:
IMO the naming duplication here isn't great, or having the input logic reach into and change state inside Game.

Can we please have this flag live inside the platform window, and be exposed via a readonly `IPlatformWindow.Minimized` property? The Game loop can then call `Renderer.WindowMinimized` which returns `Window.Minimized` the same way it proxies the other window-specific things.

Reply:
This is exactly how `Game.HasInputFocus` works just a few lines above. But I'll look into your request.

Reply:
This would be a good opportunity to fix that too :P

Reply:
Did that in #17909. This here can then use the same approach.